src/lib/features/feature-toggle/fakes/fake-feature-toggle-store.ts

```diff
@@ -323,10 +323,26 @@ export default class FakeFeatureToggleStore implements IFeatureToggleStore {
         throw new Error('Method not implemented.');
     }
 
-    getFeatureTypeCounts(
+    async getFeatureTypeCounts(
```

Comment:
added fake implementation that does simple counting

Reply:
I do not like relying on fake stores, because SQL has no much things that can go wrong and have complexity. Hmm.